packages/resolve-bus-memory/src/index.js

```diff
@@ -0,0 +1,34 @@
+const eventHandlersMap = new Map();
+
+export default function inMemoryBus() {
+    return {
+        onEvent: (eventTypesArray, callback) => {
+            eventTypesArray.forEach((eventType) => {
+                const callbackArray = eventHandlersMap.get(eventType) || [];
+                callbackArray.push(callback);
+                eventHandlersMap.set(eventType, callbackArray);
+            });
+
+            return () => {
+                eventTypesArray.forEach((eventType) => {
+                    const callbackArray = eventHandlersMap.get(eventType) || [];
+                    const callbackIndex = callbackArray.indexOf(callback);
+
+                    if (callbackIndex >= 0) {
```

Comment:
In general case, no. User can invoke `unsubscribe` function several times, so this callback can be removed from array.